gateway/src/main/java/io/camunda/zeebe/gateway/impl/broker/request/BrokerRequest.java

```diff
@@ -36,6 +37,8 @@ public BrokerRequest(final int schemaId, final int templateId) {
     this.templateId = templateId;
   }
 
+  public abstract Optional<Integer> addressesSpecificBroker();
```

Comment:
:wrench: I prefer to have the following interface
```
boolean addressSpecificBroker();
int getBorkerId();
```

You can already give default implementations of this in `BrokerRequest`. So only the requests that need to address specificBroker have to override the implementations.

Reply:
That's how I started (it's the same for addressing partitions) but then I couldn't find any reason to not just have a single method for this. I can still give a default implementation in BrokerRequest that just returns `Optional.empty()`

Is there a specific reason you prefer that interface?

Reply:
I don't have a strong opinion on this. If only one method then may be rename `addressSpecificBroker` to something like `getBroker`?

Default implementation would be good either way.

Reply:
@deepthidevaki Updated the method name and added a default implementation